projects/schematics/src/migrations/5_0/methods-and-properties-deprecations/data/methods-and-properties.migration.ts

```diff
@@ -1339,13 +1770,118 @@ export const METHODS_AND_PROPERTIES_DEPRECATIONS_DATA: MethodPropertyDeprecation
       comment:
         "// TODO:Spartacus - Method 'openModal' was removed from Class 'AddToCartComponent'. ",
     },
+    {
+      class: 'AddToWishListComponent',
+      importPath: '@spartacus/storefront',
+      deprecatedNode: 'getProductInWishList',
+      comment:
+        "The 'getProductInWishList' method's signature changed to: 'getProductInWishList( product: Product entries: OrderEntry[]): OrderEntry | undefined'",
+    },
     {
       class: 'AddToWishListComponent',
       importPath: '@spartacus/storefront',
       deprecatedNode: 'wishListService',
       comment:
         "// TODO:Spartacus - Property 'wishListService' was removed from Class 'AddToWishListComponent'. ",
     },
+    {
+      class: 'AppliedCouponsComponent',
+      importPath: '@spartacus/storefront',
+      deprecatedNode: 'cartVoucherService',
+      comment:
+        "The type of property 'cartVoucherService' changed to: 'cartVoucherService: CartVoucherFacade' ",
+    },
+    {
+      class: 'CartCouponComponent',
+      importPath: '@spartacus/storefront',
+      deprecatedNode: 'activeCartService',
+      comment:
+        "The type of property 'activeCartService' changed to: 'activeCartService: ActiveCartFacade' ",
+    },
+    {
+      class: 'CartCouponComponent',
+      importPath: '@spartacus/storefront',
+      deprecatedNode: 'cartVoucherService',
+      comment:
+        "The type of property 'cartVoucherService' changed to: 'cartVoucherService: CartVoucherFacade' ",
+    },
+    {
+      class: 'CartDetailsComponent',
+      importPath: '@spartacus/storefront',
+      deprecatedNode: 'activeCartService',
+      comment:
+        "The type of property 'activeCartService' changed to: 'activeCartService: ActiveCartFacade' ",
+    },
+    {
+      class: 'CartDetailsComponent',
+      importPath: '@spartacus/storefront',
+      deprecatedNode: 'selectiveCartService',
+      comment:
+        "The type of property 'selectiveCartService' changed to: 'selectiveCartService: SelectiveCartFacade' ",
+    },
+    {
+      class: 'CartItemComponent',
+      importPath: '@spartacus/storefront',
+      deprecatedNode: 'isProductOutOfStock',
+      comment:
+        "The 'isProductOutOfStock' method's signature changed to: 'isProductOutOfStock( product: any): any'",
```

Comment:
this didn't change at all

Reply:
I agree.  According to the api extractor, the return type changed from boolean to any.
Again, this is a case of implicit function return types that causes a debatable result.

### Method isProductOutOfStock changed.


Previous version: 

```

isProductOutOfStock(
  product: any
): boolean

```


Current version: 

```

isProductOutOfStock(
  product: any
): any

```

Reply:
I validated the data and this result comes out of the api extractor output as is.

This class moved from storefrontlib to the cart library.  Could there be differences in the compilation options that result in a different interpretation from thee api extractor?

Reply:
Edge case,  ideally, we remove these cases from the breaking change list, but gif they make it into the migration doc it's ok.

Reply:
Will push a small PR to add a type on that method.  #15478